src/org/jetbrains/plugins/scala/project/Platform.scala

```diff
@@ -23,6 +24,8 @@ object Platform {
 
   final object Dotty extends Platform("Dotty", PlatformProxy.Dotty)
 
+  final object Hydra extends Platform("Hydra", PlatformProxy.Hydra)
```

Comment:
This introduces a new case object that as to be accounted for in every `match` on the Platform. I saw a number of warnings of pattern matches that don't treat this new case, you need to make sure this is needed/checked on every other use.